src/core/runtime.d

```diff
@@ -313,13 +313,54 @@ struct Runtime
     }
 
 
+    /**
+     * Set source file path for coverage reports.
+     *
+     * Params:
+     *  path = The new path name.
+     */
+    static void coverSourcePath(string path)
```

Comment:
Yes, I was wondering why. Thanks for the explanation. But I don't think it justifies creating yet more names. I really, really do not want Phobos/Druntime to become a bloated swamp of same-only-different names. We've been through this discussion of "arcane" names before, and the pros and cons of endless renamings. Let's stop doing it.